api/overview/azure/latest/storage.md

```diff
@@ -70,6 +70,25 @@ Use the following version 25.x.x library to work with the Azure Storage resource
 
 This section details known issues for the Azure Storage client libraries for .NET.
 
+### Client-side encryption bug when using `BlobClient.UpdateClientSideEncryptionKey` with an unauthenticated symmetric encryption algorithm
+
+#### Issue description
+
+A bug was found in `BlobClient.UpdateClientSideEncryptionKey` that affects customers who are using an unauthenticated symmetric encryption algorithm. If a customer using an [impacted version](#issue-details) of **Azure.Storage.Blobs** attempts to rotate an unauthenticated symmetric encryption key using `BlobClient.UpdateClientSideEncryptionKey`, the key will not rotate correctly and decryption with the SDK might not be possible without additional steps. For remediation guidance, see [Recommended steps](#recommended-steps).
```

Comment:
A few corrections:

- "that *may* affect customers who are using an unauthenticated symmetric encryption algorithm." There's still plenty that can cause a fail-fast.
- Do we want to specify "silently rotate incorrectly" rather than just "not rotate correctly"?
- We should probably specify that this is the encryption algorithm for key-wrapping, rather than blob content encryption. They *should* know this already but when talking on this topic I find specifying helps.